src/ucs/configure.m4

```diff
@@ -83,6 +83,20 @@ AS_IF([test "x$enable_backtrace_detail" = xyes],
 )
 
 
+#
+# Disable overriding sigaction
+#
+AC_ARG_ENABLE([sigaction-override],
+	AS_HELP_STRING([--disable-sigaction-override],
+                   [Disable sigaction,signal function overriding, default: NO]),
+	[],
+	[enable_sigaction_override=yes])
+
+AS_IF([test "x$enable_sigaction_override" = xyes],
+		AC_DEFINE([ENABLE_SIGACTION_OVERRIDE], [1], [Enable sigaction,signal function overriding])
```

Comment:
pls fix indent (use spaces only) so like this:
```
AS_IF([test "x$enable_sigaction_override" = xyes],
      AC_DEFINE([ENABLE_SIGACTION_OVERRIDE], [1], [Enable sigaction,signal function overriding])
```

Reply:
@yosefe  used TAB in the above diff to be consistent with the indent used in this file.  is suggestion is to use spaces only before AC_DEFINE or everywhere in the above diff?

Reply:
pls use spaces for all new code that you add in this PR